lib/widgets/emoji_reaction.dart

```diff
@@ -160,8 +161,8 @@ class ReactionChip extends StatelessWidget {
       //   // 'Chris、Greg、Alya、Shu'
       ? userIds.map((id) {
           return id == store.selfUserId
-            ? 'You'
-            : store.users[id]?.fullName ?? '(unknown user)'; // TODO(i18n)
+            ? zulipLocalizations.reactedEmojiSelfUser
```

Comment:
Yeah, this name sounds good (re https://github.com/zulip/zulip-flutter/pull/1148#discussion_r1938033435 ).